org.uqbar.project.wollok/src/org/uqbar/project/wollok/interpreter/nativeobj/WollokJavaConversions.xtend

```diff
@@ -44,58 +49,87 @@ class WollokJavaConversions {
 			return ((o as WollokObject).getNativeObject(SET) as JavaWrapper<Set>).wrapped
 		if (o.isNativeType(BOOLEAN) && (t == Boolean || t == Boolean.TYPE))
 			return ((o as WollokObject).getNativeObject(BOOLEAN) as JavaWrapper<Boolean>).wrapped
-		
+		if (o.isNativeType(DATE)) {
+			return (o as WollokObject).getNativeObject(DATE)
+		}
+
 //		if (t.array && t.componentType == Object) {
 //			val a = newArrayOfSize(1)
 //			a.set(0, o)
 //			return a
 //		}
-			
 		if (t == Collection || t == List)
 			return #[o]
-		
+
 		// remove this ?
-		if (t.primitive)
-			return o	
-		
+		if (t.
+			primitive)
+			return o
+
 		throw new RuntimeException('''Cannot convert parameter "«o»" of type «o.class.name» to type "«t.simpleName»""''')
 	}
-	
+
 	def static dispatch isNativeType(Object o, String type) { false }
+
 	def static dispatch isNativeType(Void o, String type) { false }
+
 	def static dispatch isNativeType(WollokObject o, String type) { o.hasNativeType(type) }
-	
+
 	def static WollokObject javaToWollok(Object o) {
-		if (o == null) return null
+		if(o == null) return null
 		convertJavaToWollok(o)
 	}
-	
+
 	def static dispatch WollokObject convertJavaToWollok(Integer o) { evaluator.getOrCreateNumber(o.toString) }
+
 	def static dispatch WollokObject convertJavaToWollok(Double o) { evaluator.getOrCreateNumber(o.toString) }
+
 	def static dispatch WollokObject convertJavaToWollok(BigDecimal o) { evaluator.getOrCreateNumber(o.toString) }
+
 	// cache strings ?
 	def static dispatch WollokObject convertJavaToWollok(String o) { evaluator.newInstanceWithWrapped(STRING, o) }
+
 	def static dispatch WollokObject convertJavaToWollok(Boolean o) { evaluator.booleanValue(o) }
+
 	def static dispatch WollokObject convertJavaToWollok(List o) { evaluator.newInstanceWithWrapped(LIST, o) }
+
 	def static dispatch WollokObject convertJavaToWollok(Set o) { evaluator.newInstanceWithWrapped(SET, o) }
+
+	def static dispatch WollokObject convertJavaToWollok(LocalDate o) { evaluator.newInstanceWithWrapped(DATE, o) }
+
 	def static dispatch WollokObject convertJavaToWollok(WollokObject it) { it }
-	
-	def static dispatch WollokObject convertJavaToWollok(Object o) { 
+
+	def static dispatch WollokObject convertJavaToWollok(Object o) {
 		throw new UnsupportedOperationException('''Unsupported convertion from java «o» («o.class.name») to wollok''')
 	}
-	
+
+	/**
+	 * Este método envuelve cualquier objeto wrappeable a un WollokObject, inspeccionando sus variables de instancia
+	 */
```

Comment:
Is this being used at the end ?
I thought it was an intermediate step later not being used.
Anyway, this depends on the fact that the given "Object" is actually an instance of a Native wollok class, right ?
Because it calls evaluater.newInstance(...) passing that class of the object.
Maybe that can be documented in the javadoc (?)